client/tests/integration/unstable_network.rs

```diff
@@ -95,14 +97,11 @@ fn unstable_network(
             )),
         );
         iroha_client
-            .submit(mint_asset)
+            .submit_blocking(mint_asset)
```

Comment:
I think that's the reason why this test are flaky. Also I can't see why client shouldn't subscribe for events (blocking strategy) instead of just waiting?

But I'll bring `sleep` back